Add Features

stkyrice opened the pull request:
1.  UI to brows for CSV
2.  UI to browse for Log File destination
3.  Check for OU before creating
4.  Column name added in CSV  OUName

stkyrice committed:
Add Features

1.  UI to brows for CSV
2.  UI to browse for Log File destination
3.  Check for OU before creating
4.  Column name added in CSV  OUName

mjpagan reviewed (changes requested):
We should remove/comment the Add-ADGroupMember "Domain Admins" $_.samAccountName} until we add it to the CSV and add if not exist, then create logic.

stkyrice committed:
New-ADUser

Added Colors to the script
Fixed logic in new-aduser creation to skip and log already existing
users.
Added function to create nested OUs from CSV
